src/views/ContactenverzoekenView.vue

```diff
@@ -2,23 +2,15 @@
   <div class="klant-panel">
     <utrecht-heading :level="1">Contactverzoeken</utrecht-heading>
     <contactverzoeken-zoeker>
-      <template #onderwerp="{ contactmomentUrl }">
-        <contactmoment-details-context :url="contactmomentUrl">
-          <template #details="{ details }">
-            {{ details?.vraag || details?.specifiekeVraag }}
-          </template>
-        </contactmoment-details-context>
-      </template>
-      <template #contactmoment="{ url }">
-        <contactmoment-preview :url="url" />
+      <template #object="{ object }">
```

Comment:
sowieso veel beter dan het was. 👍
een object is altijd een zaak? misschien explicieter maken.

nb. wat als er meerdere zaken gekoppeld zijn? ondersteunen we dat bewust niet, of is dat bij een eerder refactoring per ongeluk eruit gehaald?

Reply:
Thanks! Dit slot wordt herhaald voor alle gekoppelde zaken. Dan krijg je dus meerdere keren onder elkaar: `zaaknummer: {zaak.nummer}`.